Dockerfile-alpine

```diff
@@ -0,0 +1,56 @@
+# !!! Don't try to build this Dockerfile directly, run it through bin/build-docker.sh script !!!
+FROM node:20.15.1-alpine
+
+# Configure system dependencies
+RUN apk add --no-cache --virtual .build-dependencies \
+    autoconf \
+    automake \
+    g++ \
+    gcc \
+    libtool \
+    make \
+    nasm \
+    libpng-dev \
+    python3 
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Bundle app source
+COPY . .
+
+COPY server-package.json package.json
+
+# Copy TypeScript build artifacts into the original directory structure.
+RUN ls
+RUN cp -R build/src/* src/.
```

Comment:
These can be done with one `RUN` command. It will have less layers with less commands. You can take a look on `Dockerfile`.

Reply:
I've always found it hard to read and debug when everything is in a single `RUN` command, especially as Docker caches each layer, but it looks like you're right and that it is best practices [here](https://docs.docker.com/build/building/best-practices/#run) TIL